bluebrain/repo-bluebrain/packages/spykfunc/package.py

```diff
@@ -48,19 +48,21 @@ class Spykfunc(PythonPackage):
     depends_on('py-h5py', type=('build', 'run'), when='@:0.15.1,0.16.99:')
     depends_on('py-hdfs', type=('build', 'run'))
     depends_on('py-jprops', type=('build', 'run'))
-    depends_on('py-lazy-property', type=('build', 'run'))
     depends_on('py-libsonata', type='run', when='@0.15.3:')
     depends_on('py-lxml', type=('build', 'run'))
     depends_on('py-morphio', type=('build', 'run'), when='@0.17.2:')
     depends_on('py-morpho-kit', type=('build', 'run'), when='@0.14.4:')
     depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-packaging', type=('build', 'run'), when='@0.17.3:')
     depends_on('py-pandas', type=('build', 'run'))
-    depends_on('py-progress', type=('build', 'run'))
     depends_on('py-pyarrow+parquet@3.0.0:', type=('build', 'run'))
     depends_on('py-pyspark@3.0.0:', type=('build', 'run'))
-    depends_on('py-six', type=('build', 'run'), when='@0.15.7:')
 
+    # old dependencies
     depends_on('py-bb5', type=('build', 'run'), when='@:0.15.6')
+    depends_on('py-lazy-property', type=('build', 'run'), when='@:0.16')
+    depends_on('py-progress', type=('build', 'run'), when='@:0.16')
+    depends_on('py-six', type=('build', 'run'), when='@0.15.7:0.16')
```

Comment:
Are you sure these three dependencies are not needed for versions: `'0.17.2'`, `'0.17.1'` and `'0.17.0'`?

Reply:
Yes